.travis.yml

```diff
@@ -38,11 +37,9 @@ jobs:
       script:
         - make tidy
         - git diff --exit-code
-        - go mod download
         - echo 'travis_fold:start:go.mod.graph' && echo 'go mod graph' && go mod graph && echo 'travis_fold:end:go.mod.graph'
         - pushd .. && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.6 && popd
         - make lint           # run a bunch of code checkers/linters in parallel
-        - make test-build     # build the project
```

Comment:
?

Reply:
The make test-build was to build the sidecar, which is no longer needed because the healthcheck will build it

Reply:
Still, I'd rather make the CI more obvious. If I had a crystal ball, I'm pretty sure it would show me us going crazy in the future because we don't know where the sidecar is being built.

Reply:
I am going to revert this. If the sidecar fails building, we want the build system to tell us.